Makefile

```diff
@@ -152,7 +152,7 @@ ubuntu_install: all setup_dkms udev_install dbus_install
 	@make --no-print-directory -C lib install DESTDIR=$(DESTDIR)
 	@make --no-print-directory -C daemon install DESTDIR=$(DESTDIR)
 	@make --no-print-directory -C daemon_controller install DESTDIR=$(DESTDIR)
-	
```

Comment:
The commit and related PR would be cleaner/easier to review without all these unrelated whitespace fixes.